styles/pages/TorchboxAcademy.module.scss

```diff
@@ -0,0 +1,49 @@
+.container {
```

Comment:
**Suggestion**:  Try to use SCSS varaibles ($site-width, $content--narrow) when possible.

Reply:
Thank you - I've been copying some of this code over from other pages, will update this as well. Might be useful as a utility class...

Reply:
Yeah good idea, I think a utility class would work well here and this is the ideal case for one 👍